jobs/JGLOBAL_LAND_ANALYSIS_FINALIZE

```diff
@@ -8,19 +8,23 @@ source "${HOMEgfs}/ush/jjob_header.sh" -e "landanlfinal" -c "base landanl landan
 ##############################################
 # Set variables used in the script
 ##############################################
+# Ignore possible spelling error (nothing is misspelled)
+# shellcheck disable=SC2153
 GDATE=$(date +%Y%m%d%H -d "${PDY} ${cyc} - ${assim_freq} hours")
+gPDY=${GDATE:0:8}
+gcyc=${GDATE:8:2}
 GDUMP="gdas"
 
 ##############################################
 # Begin JOB SPECIFIC work
 ##############################################
+# Generate COM variables from templates
+YMD=${PDY} HH=${cyc} generate_com -rx COM_ATMOS_ANALYSIS
```

Comment:
I think we talked about earlier in the week keeping a land analysis directory (for things like diagnostics, increments, etc.)?

Reply:
As suggested, I added the `COM_LAND_ANALYSIS` to keep a land analysis directory for the land related increments, diagnostics, etc.